vault/expiration.go

```diff
@@ -490,17 +490,23 @@ func (m *ExpirationManager) Stop() error {
 func (m *ExpirationManager) Revoke(ctx context.Context, leaseID string) error {
 	defer metrics.MeasureSince([]string{"expire", "revoke"}, time.Now())
 
+	// Setting context to the running active context
+	ctx = m.quitContext
+
 	return m.revokeCommon(ctx, leaseID, false, false)
 }
 
 // LazyRevoke is used to queue revocation for a secret named by the given
 // LeaseID. If the lease was not found it returns nil; if the lease was found
 // it triggers a return of a 202.
-func (m *ExpirationManager) LazyRevoke(leaseID string) error {
+func (m *ExpirationManager) LazyRevoke(ctx context.Context, leaseID string) error {
 	defer metrics.MeasureSince([]string{"expire", "lazy-revoke"}, time.Now())
 
+	// Setting context to the running active context
+	ctx = m.quitContext
```

Comment:
I don't think it's necessary to overwrite the context here, the request context should be sufficient since we are just marking it for revocation